src/ERC1155Mappings/premintMappings.ts

```diff
@@ -18,3 +18,21 @@ export function handlePreminted(event: Preminted): void {
 
   premint.save();
 }
+
+export function handlePremintedV2(event: PremintedV2): void {
```

Comment:
is there any difference between how we want to handle `Preminted` and `PremintedV2` ? cc @oveddan

Reply:
subgraph does not allow multiple event types in the param unfortunately - that is why i had to break it out into two separate events

Reply:
Assemblyscript doesn't really like union types.